simple/msg_sockets.c

```diff
@@ -498,7 +497,8 @@ static int setup_handle(void)
 	aihints.ai_flags = AI_PASSIVE;
 	ret = getaddrinfo(opts.src_addr, opts.src_port, &aihints, &ai);
 	if (ret == EAI_SYSTEM) {
-		FT_PRINTERR("getaddrinfo", -ret);
+		fprintf("getaddrinfo for %s:%s: %s\n",
+				opts.src_addr, opts.src_port, strerror(errno));
```

Comment:
FYI - this generates an error.  fprintf() needs a file as the first parameter.  I'll provide a fix for this.